src/clj/harja/palvelin/integraatiot/api/varusteet.clj

```diff
@@ -171,14 +171,15 @@
         muunnettu-vastausdata
         {:varusteet []}))))
 
+;; FIXME: Käyttää vanhoja VKM-muunnoksen parametreja. Päivitettävä, jos tätä käytetään vielä.
```

Comment:
Miten tälle kävi?

Reply:
Tämä on tarkoituksella jätetty. Liittyy siis siihen, että seuraamme jatkossa, käytetäänkö näitä rajapintoja vai eikö. Nykyinen oletus on, että ei käytetä, mutta haluttiin seurata vielä hetken aikaa ennen kuin poistetaan ne.